pandas/core/resample.py

```diff
@@ -1357,20 +1365,51 @@ def __init__(
         self.convention = convention or "E"
         self.convention = self.convention.lower()
 
-        if isinstance(loffset, str):
-            loffset = to_offset(loffset)
-        self.loffset = loffset
-
         self.how = how
         self.fill_method = fill_method
         self.limit = limit
-        self.base = base
+
+        self.origin = Timestamp(origin) if origin is not None else None
```

Comment:
this must be tz aware if the resample index is right? where is this checked (and tested)? mention this in the doc-string.

Reply:
Yeah you are right, otherwise it could have worked but with potential confusion for the user.
This is resolved on my last commit. I have added a test on this case and a check is perform in `_get_timestamp_range_edges`.